package.json

```diff
@@ -20,10 +20,12 @@
     "url": "git+https://github.com/coveo/react-vapor.git"
   },
   "scripts": {
-    "start": "gulp docs && webpack-dev-server --devtool eval --inline --hot --progress --colors",
-    "docs": "gulp docs",
+    "start": "npm run docs && webpack-dev-server --mode development --config webpack.config.js --devtool eval --inline --hot --progress --colors",
+    "docs": "webpack --config webpack.config.js",
```

Comment:
`webpack` implique qu'on aille webpack 4 installer globalement sur notre ordi, je propose d'essayer d'utiliser `npx webpack --config webpack.config.js` https://www.npmjs.com/package/npx

Reply:
ou `node node_modules/pathToWebpackCommand`

Reply:
I'll check, but I did not upgrade my local version of webpack for it to be updated, simply upgrade the version in package.json. What you're saying makes sense, but I'm wondering why it works anyway.

Reply:
You are right! TIL https://docs.npmjs.com/cli/run-script#description --> "Any binaries provided by locally-installed dependencies can be used without the node_modules/.bin prefix." This is very good to know :)